pom.xml

```diff
@@ -25,7 +25,7 @@
     <groupId>com.ning.billing</groupId>
     <artifactId>recurly-java-library</artifactId>
     <packaging>jar</packaging>
-    <version>0.29.1.2-SNAPSHOT</version>
+    <version>0.34.1.2-SNAPSHOT</version>
```

Comment:
The last release was 0.34, so I expect our new version to be 0.34.0.1, which will become 0.34.0.2-SNAPSHOT after we have built the released jar.